README.md

```diff
@@ -100,6 +100,7 @@ Targeting at openness and advancing state-of-art technology, [Microsoft Research
 We encourage researchers and students leverage these projects to accelerate the AI development and research.
 
 ## **Install & Verify**
+Windows local mode installation please check [NNI Windows local mode](docs/en_US/WindowsLocalMode.md)  
```

Comment:
I think it not clear that you put this sentence here. Could you split the installation for windows and linux?

Reply:
have changed